app/views/distributions/index.html.erb

```diff
@@ -96,7 +96,7 @@
                 <th>Partner</th>
                 <th class="date">Date of Distribution</th>
                 <th>Source Inventory</th>
-                <th class="numeric">Total items</th>
+                <th class="numeric">Total Number of Items</th>
```

Comment:
Could you add similiar logic to render the Item name like suggested in https://github.com/rubyforgood/human-essentials/pull/3174/files#r1008759057

Reply:
with the other PR merged in, this has been taken care already.